test/test_corpus.cc

```diff
@@ -105,10 +105,11 @@ class CFGCollectorAndTyper {
     }
 };
 
-UnorderedSet<string> knownExpectations = {
-    "parse-tree",     "parse-tree-json",    "ast",       "ast-raw",       "dsl-tree",     "dsl-tree-raw",
-    "symbol-table",   "symbol-table-raw",   "name-tree", "name-tree-raw", "resolve-tree", "resolve-tree-raw",
-    "flattened-tree", "flattened-tree-raw", "cfg",       "cfg-json",      "autogen",      "document-symbols"};
+UnorderedSet<string> knownExpectations = {"parse-tree",     "parse-tree-json",    "ast",          "ast-raw",
+                                          "dsl-tree",       "dsl-tree-raw",       "symbol-table", "symbol-table-raw",
+                                          "name-tree",      "name-tree-raw",      "resolve-tree", "resolve-tree-raw",
+                                          "flattened-tree", "flattened-tree-raw", "cfg",          "cfg-json",
+                                          "autogen",        "document-symbols",   "code-actions"};
```

Comment:
We got rid of code actions exp files, right?

Reply:
ack, missed this spot.